backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/lsm/LiveMigrateDiskCommand.java

```diff
@@ -503,7 +501,7 @@ private void updateImagesInfo() {
             DiskImageDynamic diskImageDynamic = diskImageDynamicDao.get(image.getImageId());
 
             // Update image's actual size in DB
-            if (imageFromIRS != null && diskImageDynamic != null) {
+            if (diskImageDynamic != null) {
                 diskImageDynamic.setActualSize(imageFromIRS.getActualSizeInBytes());
                 diskImageDynamicDao.update(diskImageDynamic);
             }
```

Comment:
I agree that this changes are less important. I don't know why you have to do them now, this only can delay the actual change. It is better to defer such changes which are not required for the actual feature and do them after we merge the actual feature.

Reply:
yeah, formatting is ok, but this requires thinking why it's done

Reply:
but this can also mean that a null-check is missing at line 500 ;)

Reply:
Removed this commit from the chain of this PR.
Have another PR ready with these, and some more minor fixes in `LiveMigrateDiskCommand.java` (including @ahadas suggestion to check `null` at the correct place to fix the potential NPE).
Have some doubts whether to publish it, maybe will publish later and worst case will abandon it.